### DISCUSSION TITLE
Welcome to sekai-stickers Discussions!

### DISCUSSION THREAD

**Discussion by TheOriginalAyaka** [TheOriginalAyaka/sekai-stickers#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mangorifo** [comment]


**Reply by TheOriginalAyaka** [reply]
cringe

**Reply by mangorifo** [reply]
> cringe

no u

**Comment by cheongSn** [comment]
Hope to add the second row and vertical writing words, thanks.

**Reply by TheOriginalAyaka** [reply]
hey, you can actually do that by playing with the spacing sider
![image](https://user-images.githubusercontent.com/92409239/196952959-32b0fb4c-a935-4ac6-8f43-42fceee1a780.png)

**Reply by cheongSn** [reply]
I see, thank you.

**Comment by That1Blade** [comment]
What font did you use for the sticker maker?

**Reply by TheOriginalAyaka** [reply]
Here:
https://github.com/TheOriginalAyaka/sekai-stickers/blob/main/src/fonts/YurukaStd.woff2

**Reply by That1Blade** [reply]
Thank you

**Reply by That1Blade** [reply]
Sorry to ask but do have the file in a ttf or otf format? the convertors aren't working.

**Comment by Marie-OP** [comment]
Um hey, I cleaned up a stamp recently. How would I go about submitting it to the site?

**Reply by TheOriginalAyaka** [reply]
You can open a pull request suggesting your edits, or if you don't know how to work with github, reply with the image here.

**Reply by Marie-OP** [reply]
I'll just put the image here, but I will try to learn how to use the site for future submissions. Thank you for your awesome site!

![export1672557784832](https://user-images.githubusercontent.com/122198396/211229885-5356d15f-5bcc-417e-87bc-c508f46b3a07.png)
